tests/integration/Form/ApiDataSanitiserTest.php

```diff
@@ -42,7 +42,7 @@ public function testItSanitizesBody() {
     $nestedHeading = $result[1]['body'][0];
     expect($paragraph['params']['content'])->equals('alert(1);Paragraph');
     expect($paragraph['params']['align'])->equals('left');
-    expect($nestedHeading['params']['content'])->equals('alert(1);Heading');
+    expect($nestedHeading['params']['content'])->equals('alert(2);Heading');
```

Comment:
I guess this commit "fixup! Add service for fixing form API data" should be squashed, right?